drivers/rke/rke_driver.go

```diff
@@ -52,6 +52,7 @@ type Store interface {
 	// Add methods here to get rke driver specific data
 	GetAddonTemplates(k8sVersion string) map[string]interface{}
 	GetServiceOptions(k8sVersion string) *v3.KubernetesServicesOptions
+	GetWindowsServiceOptions(k8sVersion string) *v3.KubernetesServicesOptions
```

Comment:
I think it'd make sense to combine of them in one. The return type could be `map[string]*v3.KubernetesServicesOptions` with `"k8s-service-options"` and `"k8s-windows-service-options"` as key. 

This way, when we read it here, https://github.com/rancher/rke/blob/master//cluster/plan.go#L444, RKE changes would get easier.

Reply:
Updated